cs/cs/Serializer/VowpalWabbitJsonSerializer.cs

```diff
@@ -24,33 +24,270 @@ namespace VW.Serializer
     /// </summary>
     public sealed class VowpalWabbitJsonSerializer : IDisposable
     {
-        internal const string FeatureIgnorePrefix = "_";
+        /// <summary>
+        /// Base class for JSON deserialization result.
+        /// </summary>
+        public abstract class VowpalWabbitExamples : IDisposable
```

Comment:
I was unhappy too. Renamed to VowpalWabbitExampleCollection and moved to separate files.